coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -85,18 +86,21 @@ def parse(self):
                 ' has not been implemented yet'.format(self))
 
     def _parse_documentation_with_symbols(self, param_identifiers,
+                                          excep_identifiers,
```

Comment:
Please use snake_case for variables, args, etc, which properly done excludes invented words abbreviating/shortening real words.

See https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments which recommends against "abbreviation or spelling corruption".

Of course it is never that simple.  e.g. `param_` and `doc_` are acceptable, despite being a shortened word. ;-)

A good way to catch yourself, is look if your shortened word has been used elsewhere in the codebase. i.e. `git grep excep_`.  If not, beware, you are attempting to create a new 'custom', possibly inconsistent with the name used elsewhere for the same concept.

But with "exception", I mentioned this in your previous review: https://github.com/coala/coala/pull/4281#discussion_r119154335 , where you used `excps` instead.

This shouldnt be necessary -- each review / re-submit cycle should include a fresh review by you, ensuring your entire patch has fixed the issues identified in previous reviews, especially on the same PR.  Understand and internalise these coding standards.  They are your bible ;-)